README.md

```diff
@@ -1,6 +1,10 @@
 # Applications Menu
 [![l10n](https://l10n.elementary.io/widgets/wingpanel/applications-menu/svg-badge.svg)](https://l10n.elementary.io/projects/wingpanel/applications-menu)
 
+The lightweight and stylish app launcher from elementary.
```

Comment:
I don't think you need to call out elementary here since it's already self-evident that we make the software on our org page